.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 fast_finish: true
 python:
   - 2.7
-  - 3.4
```

Comment:
Just to confirm, this was removed because of the missing conda packages for the newer pandas-datareader dependency? 

I'm currently assessing the internal value of testing this python version...

Reply:
@freddiev4 Can we keep this row in the matrix for running tests, but not for building conda packages?